integration_tests/pages/overview/OverviewPage.ts

```diff
@@ -188,5 +190,5 @@ export default class OverviewPage extends Page {
 
   private printThisPageLink = (): PageElement => cy.get('#print-link')
 
-  private goalCreationSuccessMessage = (): PageElement => cy.get('[data-qa=goal-creation-success-message]')
+  private goalCreationSuccessMessage = (): PageElement => cy.get('[data-qa=overview-success-message]')
```

Comment:
Very minor, we could also rename this function? It's now just a success message, rather than a goal creation success message 👍

Reply:
oh yeah, missed that! 🤦